### PR TITLE
Brakeman update

### PR DESCRIPTION
**Pull request by colinbruce**
Brakeman alerted on a small number of changes.

This PR reduces the brakeman alert to zero, this would allow us to turn on brakeman tests in CI

### CODE REVIEW

File: app/models/claims/sort.rb [hunk 1]
```diff
@@ -37,34 +37,44 @@ def sort(column, direction)
   #
 
   def nulls_at_top_asc(direction)
```
Comment (jsugarman) [inline app/models/claims/sort.rb]: not your name but perhaps this should be name sort_nulls or nulls_direction.  kind of like the NULLS text being in the method too
```
def sort_nulls(direction)
  'NULLS ' + (direction == 'asc' ? 'FIRST' : 'LAST')
end
```

File: app/controllers/messages_controller.rb [hunk 2]
```diff
@@ -34,17 +34,19 @@ def create
...
+                                                                  filename:    message.attachment_file_name,
+                                                                  x_sendfile:  true
   end
 
   private
 
+  def message
+    @message ||=  Message.find(params[:id])
+  end
   def send_email_if_required
```
Comment (jsugarman) [inline app/controllers/messages_controller.rb]: line break missing

File: app/views/user_message_statuses/index.html.haml [hunk 3]
```diff
@@ -13,4 +13,4 @@
         %td
           = user_message_status.message.created_at.strftime(Settings.date_time_format)
         %td
-          = link_to 'View claim', __send__("#{current_user.persona.class.to_s.pluralize.underscore}_claim_path", user_message_status.message.claim) + "#messages"
+          = link_to 'View claim', __send__(current_user.persona.message_claim_path, user_message_status.message.claim, anchor: 'messages')
```
Comment (jsugarman) [inline app/views/user_message_statuses/index.html.haml:16]: I'd be tempted to put a message_claim_path on the base class that does this dynamically (albeit that class names are not likely to change (though these they did once).
```
def message_claim_path
  "#{class.to_s.pluralize.underscore}_claim_path"
end
```

...unless brakeman no-likey

**Review by jsugarman — DISMISSED** [review]
barring suggestions comments

File: app/models/claims/sort.rb (hunk 1 above)
Comment (colinbruce) [reply app/models/claims/sort.rb]: How about...
```ruby
  def sort_nulls_by(direction)
    "NULLS #{direction == 'asc' ? 'FIRST' : 'LAST'}"
  end

  def sort_field_with_nulls(field, direction)
    "#{sort_field_by(field, direction)} #{sort_nulls_by(direction)}, id desc"
  end
```

File: app/controllers/messages_controller.rb (hunk 2 above)
Comment (colinbruce) [reply app/controllers/messages_controller.rb]: d'oh

File: app/views/user_message_statuses/index.html.haml (hunk 3 above)
Comment (colinbruce) [reply app/views/user_message_statuses/index.html.haml:16]: Unfortunately there isn't a base class. :( They both just inherit from `ActiveRecord::Base` and I don't fancy implementing an entire class to inherit this method!

File: app/models/claims/sort.rb (hunk 1 above)
Comment (jsugarman) [reply app/models/claims/sort.rb]: sweet 👍

**Review by jsugarman — APPROVED** [review]
👍

**Pull request merged**